tutorial/admin/setup_default_energy_efficient.sh

```diff
@@ -41,29 +41,40 @@
 # because these files are not expected to be writeable by non-admin users.
 #
 
-## TODO: not tested
+set -x
 
 # Range of frequencies for agent to use
 FREQ_MIN=1200000000
 FREQ_MAX=1700000000
 
+# Remove any existing configuration
+rm -rf /etc/geopm
+
 # Create the policy file in a location accessible to the compute node
 mkdir -p /etc/geopm
 POLICY_FILE_PATH="/etc/geopm/energy_efficient_policy.json"
 geopmagent -a energy_efficient -p $FREQ_MIN,$FREQ_MAX > $POLICY_FILE_PATH
 # This file should look similar to:
 #   {
-#     "FREQ_MIN": "1200000000",
-#     "FREQ_MAX": "1700000000"
+#     "FREQ_MIN": 1200000000,
+#     "FREQ_MAX": 1700000000
 #   }
 
-
 # Set the GEOPM configuration to use this policy file and the
 # energy efficient agent.
-geopmadmin --magic > $(geopmagent --config-default)
+echo "{\"GEOPM_AGENT\": \"energy_efficient\", \"GEOPM_POLICY\": \"$POLICY_FILE_PATH\"}" > $(geopmadmin --config-default)
 
 # This file should look similar to the following:
 # {
 #    "GEOPM_AGENT": "energy_efficient",
 #    "GEOPM_POLICY": "/etc/geopm/energy_efficient_policy.json"
 # }
+
+# Example sanity checks of the configuration
+#
+# Fixed frequency for non-GEOPM jobs stays at max
+#   > srun --reservation=plugin_freq_cap geopmread MSR::PERF_CTL:FREQ board 0
```

Comment:
Consider adding something that explains about the --reservation flag.  Otherwise I'd remove it.